src/olympia/files/cron.py

```diff
@@ -1,40 +0,0 @@
-import os
-import shutil
-
-from datetime import datetime
-
-from django.conf import settings
-
-import olympia.core.logger
-
-from olympia.files.models import FileValidation
-
-
-log = olympia.core.logger.getLogger('z.cron')
-
-
-def cleanup_extracted_file():
-    log.info('Removing extracted files for file viewer.')
-    root = os.path.join(settings.TMP_PATH, 'file_viewer')
-
-    for day in os.listdir(root):
-        full = os.path.join(root, day)
-
-        today = datetime.now().strftime('%m%d')
-
-        if day != today:
-            log.info('Removing extracted files: %s, from %sd.' % (full, day))
-
-            # Remove all files.
-            # No need to remove any caches since we are deleting files from
-            # yesterday or before and the cache-keys are only valid for an
-            # hour. There might be a slight edge-case but that's reasonable.
-            shutil.rmtree(full)
-
-
-def cleanup_validation_results():
```

Comment:
That's a scary function!

Reply:
at the time, validation would be re-run in the reviewer tools on demand if there wasn't a validation report already.

Reply:
Are you saying we should keep that function? I was part of `CRON_JOBS` (in settings_base.py) but not referenced anywhere else

Reply:
Definitely don't keep this. We depend on `FileValidation` existing in a couple places now.